### DISCUSSION TITLE
v0.13.1

### DISCUSSION THREAD

**Discussion by ErikBjare** [ActivityWatch/activitywatch#1072] [Releases]
These are the release notes for ActivityWatch version v0.13.1.

**New to ActivityWatch?** Check out the [website](https://activitywatch.net) and the [README](https://github.com/ActivityWatch/activitywatch/blob/master/README.md).

# Installation

See the [getting started guide in the documentation](https://docs.activitywatch.net/en/latest/getting-started.html).

# Downloads

 - [**Windows**](https://github.com/ActivityWatch/activitywatch/releases/download/v0.13.1/activitywatch-v0.13.1-windows-x86_64-setup.exe) (.exe, installer)
 - [**macOS**](https://github.com/ActivityWatch/activitywatch/releases/download/v0.13.1/activitywatch-v0.13.1-macos-x86_64.dmg) (.dmg)
 - [**Linux**](https://github.com/ActivityWatch/activitywatch/releases/download/v0.13.1/activitywatch-v0.13.1-linux-x86_64.zip) (.zip)

# Contributors

Thanks to everyone who contributed to this release:

@BelKed, @erikbjare

# Changelog

Changes since v0.13.0:

## 📦 aw-server-rust

*(excluded 3 less relevant [commits](https://github.com/ActivityWatch/aw-server-rust/compare/051bb64...3531ead))*

### 🐛 Fixes (4)

 - fix: fixed extra space before punctuation in Home view on desktop ([`2f3d1e8`](https://github.com/ActivityWatch/aw-webui/commit/2f3d1e8))
 - fix: fixes issue with migrating category settings to the server ([#583](https://github.com/ActivityWatch/aw-webui/issues/583)) ([`54769b7`](https://github.com/ActivityWatch/aw-webui/commit/54769b7))
 - fix: dark mode (Timeline, Activity, Raw Data) ([#575](https://github.com/ActivityWatch/aw-webui/issues/575)) ([`05c25c9`](https://github.com/ActivityWatch/aw-webui/commit/05c25c9))
 - fix: add saturday as option for 'start of week' setting ([`80db006`](https://github.com/ActivityWatch/aw-webui/commit/80db006))

**Full Changelog**: https://github.com/ActivityWatch/activitywatch/compare/v0.13.0...v0.13.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ActivityWatch/activitywatch/releases/tag/v0.13.1'>v0.13.1</a>.</em>

**Comment by felitsyn** [comment]
I installed ActivityWatch and found that it conflicts with Autohotkey.  AHK 1.1 scripts run, but AHK 2.0 scripts do not run. 

 It's possible (I haven't tried) that this issue is not present in version 0.10: https/github.com/ActivityWatch/activitywatch/issues/682

 Is it possible to fix ActivityWatch so that there are no conflicts with AHK 2?

**Reply by ErikBjare** [reply]
Why would ActivityWatch be the problem?

There is nothing that I know of that could cause problems with AHK. You'll need to provide more information.

**Reply by felitsyn** [reply]
I made a mistake, sorry, I was in a hurry.  The appearance of a subtle error in the script coincided with the installation of ActivityWatch. 

 The !\`::etc command is processed differently on different keyboard layouts due to the "\`" character.  In the Russian keyboard, even the compiled exe file does not run. It is better to write !sc029:: etc.  This has nothing to do with ActivityWatch.

**Comment by dartglo1121** [comment]
Virus total says this is a virus, is this true?
![image](https://github.com/ActivityWatch/activitywatch/assets/175272754/68e226d1-cd9d-4711-9fd2-af952ea86735)

**Reply by ErikBjare** [reply]
We frequently get false positives for recent releases.

We try to keep track and report them in: https://github.com/ActivityWatch/activitywatch/issues/181
